src/jest-reporters/eslint-check.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+/* eslint-disable amo/only-tsx-files */
+/* eslint-disable @typescript-eslint/no-var-requires */
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
> I really should have been able to address this by adding an entry into `devDependencies`...

Huh, I couldn't get it to work either.

Reply:
I don't know whether to 👍 this or to 👎 this. 😉